storage/storage-samples/storage-owner-sample/src/test/java/org/fido/iot/storage/To2StorageTest.java

```diff
@@ -111,6 +126,19 @@
       "target", "data",
       "ops").toString();
 
+  private static final String packageContent =
+      "#!/bin/bash\r\n"
+      + "filename=payload.bin\r\n"
+      + "cksum_tx=1612472339\r\n"
+      + "cksum_rx=$(cksum $filename | cut -d ' ' -f 1)\r\n"
+      + "if [ $cksum_tx -eq $cksum_rx  ]; then\r\n"
+      + "  echo \"Device onboarded successfully.\"\r\n"
+      + "  echo \"Device onboarded successfully.\" > result.txt\r\n"
+      + "else\r\n"
+      + "  echo \"ServiceInfo file transmission failed.\"\r\n"
+      + "  echo \"ServiceInfo file transmission failed.\" > result.txt\r\n"
+      + "fi\r\n";
+
```

Comment:
This test looks like it's unix-only.  Should the unit test be tagged with @EnableOnOs?

Reply:
This is the payload file that is used for successful serviceinfo file transfer. This not executed on the host machine during build.